application/server/server_smoke_test.go

```diff
@@ -796,6 +796,52 @@ func Test_SmokeSnykCodeFileScan(t *testing.T) {
 	assert.Eventually(t, checkForPublishedDiagnostics(t, testPath, 6, jsonRPCRecorder), maxIntegTestDuration, 10*time.Millisecond)
 }
 
+func Test_SmokeUncFilePath(t *testing.T) {
+	testutil.OnlyOnWindows(t, "testing windows UNC file paths")
+	loc, jsonRPCRecorder := setupServer(t)
+	c := testutil.SmokeTest(t, false)
+	c.SetSnykCodeEnabled(true)
+	cleanupChannels()
+	di.Init()
+
+	var cloneTargetDir, err = testutil.SetupCustomTestRepo(t, t.TempDir(), nodejsGoof, "0336589", c.Logger())
+	if err != nil {
+		t.Fatal(err, "Couldn't setup test repo")
+	}
+	defer func(path string) { _ = os.RemoveAll(path) }(cloneTargetDir)
```

Comment:
Not needed, `t.Tempdir` is taking care of its cleanup

Reply:
deleted it from other tests too.